keystorage/aws/src/test/java/tech/pegasys/signers/aws/AwsSecretsManagerTest.java

```diff
@@ -260,14 +270,34 @@ private void closeAwsSecretsManagers() {
     awsSecretsManagerInvalidCredentials.close();
   }
 
-  private void closeClients() {
-    closeAwsSecretsManagers();
-    closeTestSecretsManager();
+  private void createTestSecrets() throws RuntimeException {
+    secretName1WithTagKey1ValA =
+        createTestSecret(SECRET_NAME1, "tagKey1", "tagValA", SECRET_VALUE1);
+    secretName2WithTagKey1ValB =
+        createTestSecret(SECRET_NAME2, "tagKey1", "tagValB", SECRET_VALUE2);
+    secretName3WithTagKey2ValC =
+        createTestSecret(SECRET_NAME3, "tagKey2", "tagValC", SECRET_VALUE3);
+    secretName4WithTagKey2ValB =
+        createTestSecret(SECRET_NAME4, "tagKey2", "tagValB", SECRET_VALUE4);
   }
 
-  private String createSecret(final Tag tag, String secretValue)
+  private String createTestSecret(
```

Comment:
All this message is really doing now is create the tag...might be neater to move Tag create up one level and delete this method?